src/repositories/UserSessionRepository.ts

```diff
@@ -11,6 +11,7 @@ export class UserSessionRepository extends AbstractRepository<UserSessionModel>
     const session = new UserSessionModel();
     session.user = user;
     session.userId = user.id;
+    session.deviceToken = user.deviceTokens[user.deviceTokens.length - 1]
```

Comment:
why `user.deviceTokens.length - 1`? how do we know the session belongs to the last device added?

Reply:
i think it should be the last one because we only create a new session when someone logs in with some device token

Reply:
i'm not sure i understand.. why do we need to store device tokens in sessions in the first place?